examples/binary_examples/PyFstat_example_binary_mcmc_vs_grid_comparison.py

```diff
@@ -0,0 +1,296 @@
+)
+mcmcsearch.print_summary()
+
+# call some built-in plotting methods
+# these can all highlight the injection parameters, too
+print("Making MCMCSearch {:s} corner plot...".format("-".join(search_keys)))
+mcmcsearch.plot_corner(truths=signal_parameters)
+print("Making MCMCSearch prior-posterior comparison plot...")
+mcmcsearch.plot_prior_posterior(injection_parameters=signal_parameters)
+print("")
+
+print("*" * 20)
+print("Quantitative comparisons:")
+print("*" * 20)
+
+# some informative command-line output comparing search results and injection
+# get max twoF and binary Doppler parameters
+max_grid_index = np.argmax(twoF_values)
+max_grid_2F = twoF_values[max_grid_index]
+max_grid_parameters = grid_points[max_grid_index]
+
+# same for MCMCSearch, here twoF is separate, and non-sampled parameters are not included either
+max_dict_mcmc, max_2F_mcmc = mcmcsearch.get_max_twoF()
+print(
+    "Grid Search:\n\tmax2F={:.4f}\n\tOffsets from injection parameters (relative error): {:s}.".format(
+        max_grid_2F,
+        ", ".join(
+            [
+                "\n\t\t{1:s}: {0:.4e} ({2:.4g})%".format(
+                    max_grid_parameters[search_keys.index(key)]
+                    - signal_parameters[key],
+                    key,
+                    100
+                    * (
+                        max_grid_parameters[search_keys.index(key)]
+                        - signal_parameters[key]
+                    )
+                    / signal_parameters[key],
+                )
+                for key in search_keys
+            ]
+        ),
+    )
+)
+print(
+    "Max 2F candidate from MCMC Search:\n\tmax2F={:.4f}"
+    "\n\tOffsets from injection parameters (relative error): {:s}.".format(
+        max_2F_mcmc,
+        ", ".join(
+            [
+                "\n\t\t{1:s}: {0:.4e} ({2:.4g})%".format(
+                    max_dict_mcmc[key] - signal_parameters[key],
+                    key,
+                    100
+                    * (max_dict_mcmc[key] - signal_parameters[key])
+                    / signal_parameters[key],
+                )
+                for key in search_keys
+            ]
+        ),
+    )
+)
+# get additional point and interval estimators
+stats_dict_mcmc = mcmcsearch.get_summary_stats()
+print(
+    "Mean from MCMCSearch:\n\tOffset from injection parameters (relative error): {:s}"
+    "\n\tExpressed as fractions of 2sigma intervals: {:s}.".format(
+        ", ".join(
+            [
+                "\n\t\t{1:s}: {0:.4e} ({2:.4g})%".format(
+                    stats_dict_mcmc[key]["mean"] - signal_parameters[key],
+                    key,
+                    100
+                    * (stats_dict_mcmc[key]["mean"] - signal_parameters[key])
+                    / signal_parameters[key],
+                )
+                for key in search_keys
+            ]
+        ),
+        ", ".join(
+            [
+                "\n\t\t{1:s}: {0:.4e}".format(
```

Comment:
as these are given as percentages (note factor 100 in next line):
```suggestion
                "\n\t\t{1:s}: {0:.4f}%".format(
```

Reply:
I agree, I used `e` because I didn't want to run something and get four zeroes. But we just checked it makes sense this way, so I'll do it as such.